### PR TITLE
Example app features and fixes

### PR DESCRIPTION
**Pull request by pusztaienike**
**sn-react-browser:**
_docviewer:_
docviewer was unusable, because of styling problems (only 2 pages were visible, action buttons were also invisible)
there was no "Close" button on docviewer
saveWidget makes no sense here
_others:_
add Go to repo button
change header

**sn-react-calendar:**
add Go to repo button
change header

**sn-react-imagegallery:**
add Go to repo button

**sn-react-memoapp:**
add Go to repo button
change header

**sn-react-tasklist**
add Go to repo button

**sn-react-usersearch**
add Go to repo button
change header

### CODE REVIEW

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx [hunk 1]
```diff
@@ -56,10 +91,15 @@ interface HamburgerMenuProps {
 
 export const HamburgerMenu: React.FunctionComponent<HamburgerMenuProps> = (props) => {
   const classes = useStyles(props)
+  const isMobile = useMediaQueryOverride()
 
   const anchorRef = useRef<HTMLButtonElement>(null)
   const [isPopperOpen, setIsPopperOpen] = useState(false)
 
+  useEffect(() => {
+    setIsPopperOpen(!isMobile)
```
Comment (taki9) [inline packages/sn-universal-header-react/src/components/hamburger-menu.tsx:100]: why? maybe you want to close it when screen size changes?

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx [hunk 2]
```diff
@@ -38,11 +41,43 @@ const useStyles = makeStyles((theme: Theme) =>
       left: 0,
       backgroundColor: '#F6F6F6',
       border: '1px solid #E2E2E2',
+      height: `calc(100vh - ${HEADER_HEIGHT})`,
+      boxShadow: '12px 0 12px rgb(0 0 0 / 20%)',
+      width: '281px',
```
Comment (taki9) [inline packages/sn-universal-header-react/src/components/hamburger-menu.tsx:46]: why not auto?

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx [hunk 3]
```diff
@@ -84,44 +124,57 @@ export const HamburgerMenu: React.FunctionComponent<HamburgerMenuProps> = (props
   return (
     <div>
       <IconButton className={classes.iconButton} onClick={handleToggle} ref={anchorRef}>
-        {isPopperOpen ? <MenuOpenIcon className={classes.menuIconActive} /> : <MenuIcon className={classes.menuIcon} />}
+        {isPopperOpen ? <Close className={classes.menuIconActive} /> : <MenuIcon className={classes.menuIcon} />}
       </IconButton>
-      {isPopperOpen ? (
-        <Paper className={classes.popperWrapper}>
+      <Slide direction="right" in={isPopperOpen} mountOnEnter unmountOnExit>
+        <Paper className={clsx(classes.popperWrapper, { [classes.popperWrapperMobile]: isMobile })}>
```
Comment (taki9) [inline packages/sn-universal-header-react/src/components/hamburger-menu.tsx:130]: drawer would have been easier

File: packages/sn-universal-header-react/src/hooks/media-query-override.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { useMediaQuery, useTheme } from '@material-ui/core'
+
+export const useMediaQueryOverride = () => {
+  const theme = useTheme()
+  return useMediaQuery(theme.breakpoints.down('md'))
+}
```
Comment (taki9) [inline packages/sn-universal-header-react/src/hooks/media-query-override.ts:6]: dont do this, this logic only used in one component (and its name also bad)

**Review by taki9 — CHANGES_REQUESTED** [review]

File: examples/sn-react-browser/src/components/document-viewer.tsx [hunk 5]
```diff
@@ -2,45 +2,59 @@ import {
...
+import { Close } from '@material-ui/icons'
 import React from 'react'
-import { RouteComponentProps, withRouter } from 'react-router-dom'
+import { RouteComponentProps, useHistory, withRouter } from 'react-router-dom'
+
+const useAppBarStyles = makeStyles(() =>
+  createStyles({
+    appBar: {
+      backgroundColor: '#342cac',
+      color: '#FFFFFF',
```
[the remark this answers is not in the document]
Comment (pusztaienike) [reply examples/sn-react-browser/src/components/document-viewer.tsx:19]: I could not reproduce this. Can you please check again after cache delete? 🤔

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx (hunk 3 above)
Comment (pusztaienike) [reply packages/sn-universal-header-react/src/components/hamburger-menu.tsx]: Discussed: no changes needed

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx [hunk 6]
```diff
@@ -56,22 +91,17 @@ interface HamburgerMenuProps {
 
 export const HamburgerMenu: React.FunctionComponent<HamburgerMenuProps> = (props) => {
   const classes = useStyles(props)
+  const theme = useTheme()
+  const isUnderMd = useMediaQuery(theme.breakpoints.down('md'))
 
   const anchorRef = useRef<HTMLButtonElement>(null)
-  const [isPopperOpen, setIsPopperOpen] = useState(false)
+  const [isPopperOpen, setIsPopperOpen] = useState(!isUnderMd)
+  console.log(isUnderMd)
```
Comment (taki9) [inline packages/sn-universal-header-react/src/components/hamburger-menu.tsx:99]: remove console log

File: packages/sn-universal-header-react/src/components/hamburger-menu.tsx [hunk 7]
```diff
@@ -38,11 +41,43 @@ const useStyles = makeStyles((theme: Theme) =>
       left: 0,
       backgroundColor: '#F6F6F6',
       border: '1px solid #E2E2E2',
+      height: `calc(100vh - ${HEADER_HEIGHT})`,
+      boxShadow: '12px 0 12px rgb(0 0 0 / 20%)',
+      width: 'auto',
```
Comment (taki9) [inline packages/sn-universal-header-react/src/components/hamburger-menu.tsx:46]: default value of width is auto

**Review by taki9 — CHANGES_REQUESTED** [review]

**Review by herflis — APPROVED** [review]

**Review by taki9 — APPROVED** [review]

**Pull request merged**